src/lambda.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
```

Comment:
I don't understand the point of this file, why not just have everything in one script?